lib/Parser/Parse.cpp

```diff
@@ -3440,6 +3442,11 @@ ParseNodePtr Parser::ParseTerm(BOOL fAllowCall,
 
         ChkCurTok(tkRParen, ERRnoRparen);
 
+        if (looseCoalesce != nullptr)
+        {
+            *looseCoalesce = false;
```

Comment:
The last passed statement was wrapped in brackets - therefore it is not a "loose" coalesce.